actors/power/tests/power_actor_tests.rs

```diff
@@ -198,3 +201,176 @@ fn claimed_power_given_claim_does_not_exist_should_fail() {
     rt.verify();
     h.check_state();
 }
+
+fn get_enrolled_cron_ticks(rt: &MockRuntime, epoch: ChainEpoch) -> Vec<CronEvent> {
+    let state: State = rt.get_state();
+    let events_map = Multimap::from_root(
+        &rt.store,
+        &state.cron_event_queue,
+        CRON_QUEUE_HAMT_BITWIDTH,
+        CRON_QUEUE_AMT_BITWIDTH,
+    )
+    .expect("failed to load cron events");
+
+    let mut events: Vec<CronEvent> = Vec::new();
+    events_map
+        .for_each::<_, CronEvent>(&epoch_key(epoch), |_, v| {
+            events.push(v.to_owned());
+            Ok(())
+        })
+        .unwrap();
+
+    events
+}
+
+fn enroll_cron_event(
+    rt: &mut MockRuntime,
+    epoch: ChainEpoch,
+    miner_address: &Address,
+    payload: &RawBytes,
+) {
+    rt.set_caller(*MINER_ACTOR_CODE_ID, miner_address.to_owned());
+    rt.expect_validate_caller_type(vec![*MINER_ACTOR_CODE_ID]);
+    let params =
+        RawBytes::serialize(EnrollCronEventParams { event_epoch: epoch, payload: payload.clone() })
+            .unwrap();
+    rt.call::<PowerActor>(Method::EnrollCronEvent as u64, &params).unwrap();
+}
+
+#[test]
+fn enroll_cron_epoch_multiple_events() {
+    let (h, mut rt) = setup();
+
+    let peer = "miner".as_bytes().to_vec();
+    let multiaddrs = vec![BytesDe("multiaddr".as_bytes().to_vec())];
+
+    let miner1_address = Address::new_id(301);
+    h.create_miner(
+        &mut rt,
+        &OWNER,
+        &OWNER,
+        &miner1_address,
+        &ACTOR,
+        peer.clone(),
+        multiaddrs.clone(),
+        RegisteredPoStProof::StackedDRGWindow32GiBV1,
+        &TokenAmount::from(10),
+    )
+    .unwrap();
+
```

Comment:
The go test creates a miner, enrolls cron events and then creates another miner.  In this case it doesn't matter for correctness but for future PRs please keep the same control flow as the go tests for expedience of review unless there is a good reason to change it, for example when fixing bugs or removing unnecessary operations.

Same goes for data values.  Adding balance 10 or changing miner2 address to 302 from 501 doesn't hurt correctness here but it slows down pattern matching during review slightly without any benefit.  And in some cases it could invalidate tests to miss bugs the go tests do catch.